README.md

```diff
@@ -1,4 +1,7 @@
 # Mona Python SDK
+![Mona logo](https://github.com/monalabs/mona-sdk/blob/main/mona_logo.png?raw=true)
```

Comment:
Now this image won't work, because the link is pointing to the image saved in main (and its not there yet). It will work after the merge:)